### ISSUE TITLE
Video Series Demoing Violations and Fixes

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#329]
## Overview
Our video series will showcase the implications of accessibility violations for users. It'll start with me showing an issue reported in Equalify. For instance, "Invalid Heading Order." I'll talk about how we scanned for that issue and generally what it means. Then @kevinandrews1  will demo the site with the issue, speaking to his experience of why the violation is a big deal. I'll then talk about how to fix this issue, leading into @kevinandrews1 using a fixed version of the site.

## Publication
We can do one short video per common accessibility violation. The videos can be released to all our marketing channels, on our website and perhaps linked in our app. We can push for folks to subscribe to Equalify for weekly updates.

## Tasks
- [x] Kevin and I will record an initial video on 6/19/24.
- [x] Record three more videos in subsequent weeks.
- [x] After we have three videos recorded, post online.
- [ ] Continue recording videos, unless we don't see growth in subscribers.

**Comment by kevinandrews1** [comment]
Excellent @bbertucc! Do we have a mechanism for folks to subscribe to updates like a newsletter/mass email blast?

**Comment by bbertucc** [comment]
No. Have you used any newsletter services in the past? I usually use Campaign Monitor, but could use one you're familiar with @kevinandrews1

**Comment by kevinandrews1** [comment]
@bbertucc I marked recording the initial video on 6/19 (even though we did it 6/20) as complete. Re Campaign Monitor, haven't used it, though their blog has some a11y-related content around accessible emails so that seems promising. I'm open to trying whatever we've already got. Can you get me access?

**Comment by bbertucc** [comment]
I just sent you and invite @kevinandrews1 to campaign monitor. I can't speak to their support of screen readers. Poke around. If it looks good, we can actually maintain our contact list there. Also happy to walk you through it next time we chat.

**Comment by bbertucc** [comment]
Another option, @kevinandrews1, is to just feed names of people who sign up into our Google sheet. We can bypass Campaign Monitor that way.

Maybe we create a common Google account we send/receive emails from then use the "Mail Merge" feature to send to folks. Here's some documentation on that feature: https://developers.google.com/apps-script/samples/automations/mail-merge

What do you think?

**Comment by kevinandrews1** [comment]
Idk @bbertucc that sounds more complicated but not familiar with that process. Let's give Campaign Monitor a go. Gonna poke around in there today and we can discuss tomorrow on our call in the afternoon.

**Comment by bbertucc** [comment]
Going to close this since we have 3 in the can.
